pages/index.haml

```diff
@@ -17,6 +17,142 @@
   -# %p
   -#   For more information, see our
   -#   %a{:href => "#faq", :style => "display: inline"} FAQ
+
+- rally_registration_url = "https://us06web.zoom.us/webinar/register/WN__M_n2GlnT-yRfTbQ8D7lcw#/registration"
+%section.section-2#solidarity-rally
+  %header
+    %h3 Join Our Solidarity Rally: Show Your Support
+    %p.paragraph
+      We’re fighting for a fair contract at Kickstarter. We want to codify the four-day, 32-hour
+      workweek, raise the minimum salary, and protect workers from being replaced by contractors
+      or AI tools. Stand with Kickstarter United at our public virtual rally.
+
+  %article
+    %h4 Rally Details
+
+    %p
+      %strong When:
+      %time{ datetime: "2025-09-05T14:00:00-07:00" } Friday, September 5, 2025, 2:00 PM PT
+      %time{ datetime: "2025-09-05T17:00:00-04:00" } (5:00 PM ET)
+      %br/
+      %strong Where:
+      Online (registration required)
+      %br/
+      %strong Duration:
+      1 hour
+
+    %p
+      %a{ href: rally_registration_url, rel: "noopener", target: "_blank" }
+        Register for the Rally
```

Comment:
I would maybe combine the Register for the rally bit with the **Where**, e.g.

**Where**: Online (registration required). Register here!